sgx-types/src/ssa.rs

```diff
@@ -254,6 +308,20 @@ impl Footer {
             gpr,
         }
     }
+
+    /// Converts an internal fault type to a safe Rust enum describing the fault.
+    pub fn fault(&self) -> Option<Fault> {
+        match (self.gpr.exitinfo.exit_type(), self.gpr.exitinfo.exception()) {
+            (Some(_et), Some(Exception::PageFault)) => Some(Fault::PF(
```

Comment:
```suggestion
            (Some(_), Some(Exception::PageFault)) => Some(Fault::PF(
```

However, if we aren't going to use the exit type for anything, we should maybe just do this check up front. I think even just this should work...

```rust
self.gpr.exitinfo.exit_type()?;
```

Then it can be removed from the match statement to declutter it.